base/sysinfo.jl

```diff
@@ -141,12 +141,22 @@ function cpu_info()
     return cpus
 end
 
+"""
+    uptime()
+
+Gets the current system uptime in seconds.
+"""
 function uptime()
     uptime_ = Ref{Float64}()
     Base.uv_error("uv_uptime",ccall(:uv_uptime, Int32, (Ptr{Float64},), uptime_))
     return uptime_[]
 end
 
+"""
+    loadavg()
+
+Gets the load average; see: https://en.wikipedia.org/wiki/Load_(computing).
```

Comment:
Perhaps mark up the link? Not important, but the punctuation could simplify to e.g. "Get the load average. See [...](...)"?